doc/docs.md

```diff
@@ -7134,17 +7134,8 @@ To use these functions the C header for the used OS has to be included and the f
 that are intended to be used have to be declared. Example:
 
 ```v globals
-$if windows {
-	#include "@VEXEROOT/thirdparty/stdatomic/win/atomic.h"
-} $else {
-	#include "@VEXEROOT/thirdparty/stdatomic/nix/atomic.h"
-}
 
-// declare functions we want to use - V does not parse the C header
-fn C.atomic_store_u32(&u32, u32)
-fn C.atomic_load_u32(&u32) u32
-fn C.atomic_compare_exchange_weak_u32(&u32, &u32, u32) bool
-fn C.atomic_compare_exchange_strong_u32(&u32, &u32, u32) bool
+import sync as _
 
```

Comment:
The docs should stay as they are. They show a more manual low level of interoperating with C, and V's atomics should be improved instead (that is just not a current priority, since it also involves upgrading tcc and potentially adding support for atomics to it too).